backend-project/small_eod/generic/tests.py

```diff
@@ -61,4 +68,33 @@ def get_create_data(self):
         for field in self.get_ommited_fields():
             del data[field]
         del data["id"]
-        return data
\ No newline at end of file
+        return data
+
+
+class FactoryTestCase(TestCase):
```

Comment:
```suggestion
class FactoryTestCaseMixin:
```
Nie możesz dziedziczyć z TestCase, bo test-runner będzie chciał od razu załadować. Poza tym proponuje inną nazwę.